cider/tests/utils/CiderBatchStringifier.cpp

```diff
@@ -177,3 +180,30 @@ std::string ScalarBatchStringifier<bool>::stringifyValueAt(CiderBatch* batch,
         static_cast<int64_t>(CiderBitUtils::isBitSetAt(data_buffer, row_index)));
   }
 }
+
+std::string VarcharBatchStringifier::stringifyValueAt(CiderBatch* batch, int row_index) {
+  auto varchar_batch = batch->as<VarcharBatch>();
+  if (!varchar_batch) {
+    CIDER_THROW(CiderRuntimeException,
+                "ScalarBatch is nullptr, maybe check your casting?");
+  }
+
+  auto data_buffer = varchar_batch->getRawData();
+  auto offset_buffer = varchar_batch->getRawOffset();
+  auto valid_bitmap = varchar_batch->getNulls();
```

Comment:
too much auto, please use the actual type.

Reply:
Understood. But may I ask is it for better readability or for other concerns?

Reply:
we can refer [google cpp code style](https://google.github.io/styleguide/cppguide.html#Type_deduction). 
`Use type deduction only if it makes the code clearer to readers who aren't familiar with the project, or if it makes the code safer. Do not use it merely to avoid the inconvenience of writing an explicit type.`

Reply:
Ok, thanks for the explanation! Fixed.